pandas/tests/io/xml/test_xml.py

```diff
@@ -1444,7 +1444,7 @@ def test_file_io_iterparse(datapath, parser, mode):
     filename = datapath("io", "data", "xml", "books.xml")
 
     funcIO = StringIO if mode == "r" else BytesIO
-    with open(filename, mode) as f:
+    with open(filename, mode, encoding="utf-8" if mode=="r" else None) as f:
```

Comment:
still gonna have to run `pre-commit` on the files you've modifed though ;)

Reply:
i was run it. it passed all of the hooks. only black (hook) failed it